src/api/test/functional/webui/patchinfo_create_test.rb

```diff
@@ -120,7 +120,17 @@ def delete_patchinfo project
     find(:id, 'delete-patchinfo').click
     find(:id, 'del_dialog').must_have_text 'Delete Confirmation'
     find_button("Ok").click
+
+    assert_equal page.current_path, project_show_path(project)
     find('#flash-messages').must_have_text "'patchinfo' was removed successfully from project"
+
+    # FIXME: There must be a better way to test this
+    begin
+      Package.get_by_project_and_name(project.to_param, "patchinfo")
+      assert false
+    rescue Package::UnknownObjectError => e
+      assert_equal "home:Iggy/patchinfo", e.message
+    end
```

Comment:
You can use:

exception = assert_raise Package::UnknownObjectError do
...
end
assert_equal "home:Iggy/patchinfo", exception.message